doc/nrf/ug_zigbee_configuring_components.rst

```diff
@@ -0,0 +1,31 @@
+﻿.. _ug_zigbee_configuring_components:
+
+Configuring Zigbee libraries in |NCS|
+#####################################
+
+The Zigbee protocol in |NCS| can be customized by enabling and configuring several libraries.
+
+This page lists options and steps required for configuring the available libraries:
+
+.. contents::
+    :local:
+    :depth: 1
+
+.. _ug_zigbee_configuring_components_ota:
+
+Configuring default signal handler
+**********************************
+
+The default signal handler provides a default logic to handle ZBOSS stack signals.
+For more information, see :ref:`lib_zigbee_signal_handler`.
+
+The default signal handler can be enabled and configured using the following Kconfig options:
+
+* (mandatory option 1) - This option must be enabled to use the DFU OTA.
+* (optional option 1) - This option allows to do XYZ.
+* (optional option 2) - This option allows to do XYZ.
+
+When configuring default signal handler, take into account the following principles:
```

Comment:
We may rewrite it if you want to, to mention the features that are left unimplemented in default signal handler (such as parsing the production configuration, finding and binding, commissioning other than a network steering with default TCLK).

Reply:
Not sure, to be honest...